beamer/tests/agent/unit/test_invalidate.py

```diff
@@ -0,0 +1,75 @@
+from hexbytes import HexBytes
+
+from beamer.events import FillHashInvalidated, HashInvalidated
+from beamer.state_machine import process_event
+from beamer.tests.agent.unit.utils import CLAIM_ID, FILL_ID, make_claim, make_context, make_request
+from beamer.typing import ClaimId, FillId
+
+
+def test_handle_fill_hash_invalidated():
+    context, config = make_context()
+
+    request = make_request()
+    request.fill(config.account.address, b"", FILL_ID)
+    context.requests.add(request.id, request)
+    assert request.fill_id is not None
+
+    claim_1 = make_claim(request=request, claimer=config.account.address, fill_id=request.fill_id)
+    claim_2 = make_claim(
+        request=request,
+        claim_id=ClaimId(CLAIM_ID + 1),
```

Comment:
Nitpick: I'm unsure of the relation of `CLAIM_ID` and `claim_1.id` here. If they are supposed to be the same or related, then I'd suggest deriving the second claim ID from the first one, e.g. `ClaimId(claim_1.id + 1)` or similar.